addons/variable-manager/style.css

```diff
@@ -0,0 +1,111 @@
+.sa-var-manager {
+  display: block;
+  padding: 18px;
+  /* weird hack to fix scrolling??? */
+  height: 50px;
+  overflow-y: auto;
+}
+
+.sa-var-manager-searchbox {
+  border: 1px solid #dddddd;
+  background: none;
+  background-image: url("search.svg");
+  margin-bottom: 4px;
+  padding: 8px;
+  padding-right: 5%; /* for the text to not overlap the image */
+  border-radius: 4px;
+  background-repeat: no-repeat;
+  background-size: 18px 18px;
+  background-position: 95% center;
+}
+
+/* begin feedback stuff (remove when out of beta)*/
+.sa-var-manager-feedback {
+  float: right;
+}
+
+.sa-var-manager-feedback-description {
+  color: gray;
+  font-style: italic;
+}
+
+.sa-var-manager-feedback-button {
+  display: inline-block;
+  margin-left: 4px;
+  margin-right: 4px;
+  background-color: #ff7b26;
+  padding: 4px;
+  border-radius: 4px;
+  color: white !important;
+  transition: 250ms;
+}
+
+.sa-var-manager-feedback-button:hover {
+  color: white !important;
+  opacity: 0.75;
+}
+
+/* end feedback stuff */
+
+.sa-var-manager.freeze > * > * > * > * > * {
+  opacity: 0.5;
+}
+
+.sa-var-manager.freeze input:focus {
+  opacity: 1 !important;
+}
+
+.sa-var-manager.freeze textarea:focus {
+  opacity: 1 !important;
+}
+
+.sa-var-manager-heading {
+  display: block;
+  font-weight: bold;
+  font-size: large;
+}
+
+.sa-var-manager-name {
+  word-break: break-word;
+}
+
+.sa-var-manager-value {
+  width: 75%;
+  padding: 0 !important;
+}
+
+.sa-var-manager-value > input {
+  background: none;
+  border: none;
+  padding: 8px;
+  width: 100%;
+  height: 100%;
+}
+
+.sa-var-manager-value > textarea {
+  background: none;
+  border: none;
+  padding: 8px;
+  width: 100%;
+  height: 100%;
+  line-height: 2em;
+  resize: none;
+}
+
+.sa-var-manager table {
+  border-radius: 5px;
+  border-collapse: collapse;
+  width: 100%;
+}
+
+.sa-var-manager td,
+th {
+  border: 1px solid #dddddd;
+  text-align: left;
+  padding-left: 8px;
+  padding-right: 8px;
+}
+
+/* tr:nth-child(even) {
+    background-color: #dddddd;
+} */
```

Comment:
why commented out?

Reply:
because it's something i might reconsider one day?